src/journal/journald.c

```diff
@@ -52,7 +52,7 @@ int main(int argc, char *argv[]) {
                 goto finish;
 
         server_vacuum(&server, false);
-        server_flush_to_var(&server);
+        server_flush_to_var(&server, true);
```

Comment:
i have downgraded the line in the commit msg now from "fixes: " to "see: ", for the next push